pwm/install_pwm.sls

```diff
@@ -0,0 +1,108 @@
+#installs packages needed for PWM, downloads the WAR file and installs it into tomcat
+install_packages:
+  pkg.installed:
+    - names:
+      - java-1.8.0-openjdk
+      - tomcat
+      - wget
+      - unzip
+      - httpd
+      - inotify-tools
+      - s3cmd
+      - at
+      - postfix
+      - cyrus-sasl-plain
+      - chrony
+      - python-setuptools
+      - python2-pip
+
+#install_aws_cli:
+#  pip.installed:
+#    - name: awscli
+
+remove_unused_packages:
+  pkg.removed:
+    - names:
+      - ntp
+      - ntpdate
+
+retrieve_war_file:
+  cmd.run:
+    - name: aws s3 cp s3://{{ salt.pillar.get('pwm:lookup:config_bucket') }}/pwm18.war /usr/share/tomcat/webapps/ROOT.war
+
+change_war_file_ownership:
+  file.managed:
+    - name: /usr/share/tomcat/webapps/ROOT.war
+    - user: tomcat
+    - group: tomcat
+    - replace: False
+
+start_tomcat_service:
+  service.running:
+    - name: tomcat
+    - enable: True
+    - reload: True
+
+sleep_ten_seconds:
+  cmd.run:
+    - name: sleep 10
+
+restart_tomcat_service:
+  service.running:
+    - name: tomcat
+    - enable: True
+    - reload: True
+
+append_to_pwm.conf:
+  file.append:
+    - name: /etc/httpd/conf.d/pwm.conf
+    - source: salt://files/install_pwm/pwm.conf
+
+start_httpd_service:
+  service.running:
+    - name: httpd
+    - enable: True
+
+start_atd_service:
+  service.running:
+    - name: atd
+    - enable: True
+
+create_selinuxproxy_script:
```

Comment:
this script can be replaced by `selinux.boolean` and a `cmd.run` that executes `chcon -R --reference=/usr/share/tomcat/webapps /usr/share/tomcat/webapps/ROOT.war`

Reply:
Yeah, not familiar with SELinux so this is all foreign to me atm

Reply:
Looking over this script... not sure how this can be replaced by selinux.boolean and cmd.run. Please expand upon this.